boards/arduino-mega2560/include/arduino_pinmap.h

```diff
@@ -31,93 +32,78 @@ extern "C" {
-#define ARDUINO_PIN_39          GPIO_PIN(PORT_L, 4)
-#define ARDUINO_PIN_40          GPIO_PIN(PORT_L, 5)
-#define ARDUINO_PIN_41          GPIO_PIN(PORT_L, 6)
-#define ARDUINO_PIN_42          GPIO_PIN(PORT_L, 7)
-#define ARDUINO_PIN_43          GPIO_PIN(PORT_D, 0)
-#define ARDUINO_PIN_44          GPIO_PIN(PORT_D, 1)
-#define ARDUINO_PIN_45          GPIO_PIN(PORT_D, 2)
-#define ARDUINO_PIN_46          GPIO_PIN(PORT_D, 3)
-#define ARDUINO_PIN_47          GPIO_PIN(PORT_D, 4)
-#define ARDUINO_PIN_48          GPIO_PIN(PORT_D, 5)
-#define ARDUINO_PIN_49          GPIO_PIN(PORT_D, 6)
-#define ARDUINO_PIN_50          GPIO_PIN(PORT_D, 7)
-#define ARDUINO_PIN_51          GPIO_PIN(PORT_G, 0)
-#define ARDUINO_PIN_52          GPIO_PIN(PORT_G, 1)
-#define ARDUINO_PIN_53          GPIO_PIN(PORT_C, 0)
-#define ARDUINO_PIN_54          GPIO_PIN(PORT_C, 1)
-#define ARDUINO_PIN_55          GPIO_PIN(PORT_C, 2)
-#define ARDUINO_PIN_56          GPIO_PIN(PORT_C, 3)
-#define ARDUINO_PIN_57          GPIO_PIN(PORT_C, 4)
-#define ARDUINO_PIN_58          GPIO_PIN(PORT_C, 5)
-#define ARDUINO_PIN_59          GPIO_PIN(PORT_C, 6)
-#define ARDUINO_PIN_60          GPIO_PIN(PORT_C, 7)
-#define ARDUINO_PIN_63          GPIO_PIN(PORT_J, 0)
-#define ARDUINO_PIN_64          GPIO_PIN(PORT_J, 1)
-#define ARDUINO_PIN_65          GPIO_PIN(PORT_J, 2)
-#define ARDUINO_PIN_66          GPIO_PIN(PORT_J, 3)
-#define ARDUINO_PIN_67          GPIO_PIN(PORT_J, 4)
-#define ARDUINO_PIN_68          GPIO_PIN(PORT_J, 5)
-#define ARDUINO_PIN_69          GPIO_PIN(PORT_J, 6)
-#define ARDUINO_PIN_70          GPIO_PIN(PORT_G, 2)
-#define ARDUINO_PIN_71          GPIO_PIN(PORT_A, 7)
-#define ARDUINO_PIN_72          GPIO_PIN(PORT_A, 6)
-#define ARDUINO_PIN_73          GPIO_PIN(PORT_A, 5)
-#define ARDUINO_PIN_74          GPIO_PIN(PORT_A, 4)
-#define ARDUINO_PIN_75          GPIO_PIN(PORT_A, 3)
-#define ARDUINO_PIN_76          GPIO_PIN(PORT_A, 2)
-#define ARDUINO_PIN_77          GPIO_PIN(PORT_A, 1)
-#define ARDUINO_PIN_78          GPIO_PIN(PORT_A, 0)
-#define ARDUINO_PIN_79          GPIO_PIN(PORT_J, 7)
-#define ARDUINO_PIN_82          GPIO_PIN(PORT_K, 7)
-#define ARDUINO_PIN_83          GPIO_PIN(PORT_K, 6)
-#define ARDUINO_PIN_84          GPIO_PIN(PORT_K, 5)
-#define ARDUINO_PIN_85          GPIO_PIN(PORT_K, 4)
-#define ARDUINO_PIN_86          GPIO_PIN(PORT_K, 3)
-#define ARDUINO_PIN_87          GPIO_PIN(PORT_K, 2)
-#define ARDUINO_PIN_88          GPIO_PIN(PORT_K, 1)
-#define ARDUINO_PIN_89          GPIO_PIN(PORT_K, 0)
-#define ARDUINO_PIN_90          GPIO_PIN(PORT_F, 7)
-#define ARDUINO_PIN_91          GPIO_PIN(PORT_F, 6)
-#define ARDUINO_PIN_92          GPIO_PIN(PORT_F, 5)
-#define ARDUINO_PIN_93          GPIO_PIN(PORT_F, 4)
-#define ARDUINO_PIN_94          GPIO_PIN(PORT_F, 3)
-#define ARDUINO_PIN_95          GPIO_PIN(PORT_F, 2)
-#define ARDUINO_PIN_96          GPIO_PIN(PORT_F, 1)
-#define ARDUINO_PIN_97          GPIO_PIN(PORT_F, 0)
+#define ARDUINO_PIN_0		GPIO_PIN(PORT_E, 0)
```

Comment:
here and below: please use 4 space indention instead of tabs (see coding guidelines https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions)